etna/Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM etna-base
+ENV ETNA_GEM_DEVELOPMENT=1
```

Comment:
Etna binary will use bunder in make -C bash but will use gem management when installed.